src/common.c

```diff
@@ -674,8 +674,34 @@ void ofi_straddr_log_internal(const char *func, int line,
 
 int fi_epoll_create(struct fi_epoll **ep)
 {
+	int ret;
+
 	*ep = calloc(1, sizeof(struct fi_epoll));
-	return *ep ? 0 : -FI_ENOMEM;
+	if (!*ep)
+		return -FI_ENOMEM;
+
+	(*ep)->fds = calloc(64, sizeof(*(*ep)->fds) +
```

Comment:
The value `64` is repeated twice: line 683 and 694. I would suggest to define a new macro to store this value, or at least create a `const` variable set to `64` at the beginning of the function.